src/Raven.Client/Documents/Commands/DeleteDocumentCommand.cs

```diff
@@ -17,6 +17,7 @@ public DeleteDocumentCommand(string id, long? etag)
 
             _id = id;
             _etag = etag;
+            ResponseType = RavenCommandResponseType.Empty;
```

Comment:
Thanks.

I've meant RavenCommand -> Empty and RavenCommand[T] -> Object. I think that the parenthesis were stripped.